src/main/java/view_post/use_case/ViewPostResponseModel.java

```diff
@@ -0,0 +1,64 @@
+package view_post.use_case;
+
+/**
+ * A data class for storing the output data.
+ */
+public class ViewPostResponseModel {
+    private final String posterEmail;
+    private final String postBody;
+    private final String[] postTags;
+    private final String[] postReplies;
+    private final String deadline;
+
```

Comment:
Could we add creationDate (String), collaborators (String), and postID (int)?

Reply:
Will add creationDate and collaborators but do we really need postID? This is information that is being displayed to the user and I don't think that piece of info is important to them so we can leave it out. Or are we keeping the postID so we know which post to delete when deleting posts?